website/client/src/components/ui/filterSidebar.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="filter">
+    <slot name="header"></slot>
+    <slot name="search"></slot>
+    <div class="form">
+      <h2 v-once class="filter-label">
+        {{ $t('filters') }}
+      </h2>
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'filterSidebar',
+};
+</script>
+
+<style lang="scss" scoped>
+@import '~@/assets/scss/colors.scss';
+
+.filter-label {
+  font-family: Roboto, sans-serif;
```

Comment:
can we remove the font-family and letter spacing properties? they're not really necessary

Reply:
changed the h3/h4 stylings to be normal Roboto instead of Condensed so these changes will be gone after my next push